version.py

```diff
@@ -21,6 +21,10 @@
 DESKTOP_MINIMUM_VERSION = "5.0.0"
 DESKTOP_WARNING_VERSION = "5.0.0"
 
+# Bump the API_FEATURE_LEVEL whenever an API change is made
+# that clients might want to condition on.
+API_FEATURE_LEVEL = 1
```

Comment:
```suggestion
# Bump the API_FEATURE_LEVEL whenever an API change is made
# that clients might want to condition on.  If we forget at the time we
# make the change, then bump it later as soon as we notice.
API_FEATURE_LEVEL = 1
```

That's a bit of a surprising property -- and a very helpful one -- so good to make explicit.